src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -377,6 +377,10 @@ static bool ReferencesNServiceBus(Assembly assembly, List<AssemblyName> processe
                 {
                     continue;
                 }
+                catch (FileLoadException)
```

Comment:
My test runner was loading nunit v2 and v3 into the domain, which the assembly scanner was then trying to load and scan. The second version of nunit was causing this to fail with `FileLoadException`.